src/domainManagementUI/src/app/components/domain/domain-create-dialog/domain-create-dialog.component.html

```diff
@@ -1,38 +1,68 @@
-<h2 mat-dialog-title>Add New Domain</h2>
+<h2 mat-dialog-title>Add New Domains</h2>
 <button
   mat-button
   style="display: inline-block; top: -4em; margin-bottom: -4em"
   [mat-dialog-close]="false"
 ></button>
-<form [formGroup]="domainForm" class="dialog text-muted" mat-dialog-content>
+<form
+  [formGroup]="domainForm"
+  class="dialog text-muted"
+  style="height: 25em"
+  mat-dialog-content
+>
   <div class="text-muted mb-4">
-    Provide a domain and select submit to create a new domain
+    Please enter one or more domains to add, seperated by a comma.
   </div>
 
   <div class="flex-column" [formGroup]="domainForm">
     <div class="flex flex-column">
       <mat-label>Domain</mat-label>
       <mat-form-field style="width: 30em" appearance="outline">
-        <mat-label>Domain URL</mat-label>
-        <input
+        <mat-label>Domain URL's</mat-label>
+        <textarea
           matInput
           trim="blur"
           type="text"
           formControlName="url"
           placeholder=" The url of the new domain"
-        />
+        ></textarea>
```

Comment:
Can we get this to expand a bit more by default? Doesn't look that great on my screen.

![image](https://user-images.githubusercontent.com/1170974/114440408-51c9c500-9b87-11eb-9027-8722831154b4.png)

Reply:
Expanded more by default.